Amplitude - iOS v2 device mode update (#628)
rudderlabs/rudderstack-docs: docs/destinations/analytics/amplitude.mdx

## Adding device mode integration

> Once you add Amplitude as a destination in the [RudderStack dashboard](https://app.rudderstack.com/), follow these steps to add it to your mobile project depending on your integration platform:

Reviewer: ```suggestion
Once you add Amplitude as a destination in the [RudderStack dashboard](https://app.rudderstack.com/), follow these steps to add it to your project depending on your integration platform:
```

<Tabs>
  <TabList>
    <Tab>iOS</Tab>

>     <Tab>iOS V2</Tab>

Reviewer: ```suggestion
    <Tab>iOS v2</Tab>
```

    <Tab>Android</Tab>
    <Tab>React Native</Tab>
  </TabList>
    <TabPanels>
      <TabPanel>
        Follow these steps to add Amplitude to your iOS project:
<ol>

> <li>In your <code class="inline-code">Podfile</code> and add the <code class="inline-code">Rudder-Amplitude</code> extension

Reviewer: ```suggestion
<li>In your <code class="inline-code">Podfile</code>, add the <code class="inline-code">Rudder-Amplitude</code> extension, as shown:
```

```
</span>
</li>
</ol>
      </TabPanel>
        <TabPanel>
          <div class="warningBlock">
          This device mode integration is supported for Amplitude v8.8.0 and above.
        </div>
        Follow these steps to add Amplitude to your iOS project:
<ol>

> <li> Install <code class="inline-code">RudderAmplitude</code> (available through <a href="https://cocoapods.org/">CocoaPods</a>) by adding the following line to your <code class="inline-code">Podfile</code>.

Reviewer: ```suggestion
<li> Install <code class="inline-code">RudderAmplitude</code> (available through <a href="https://cocoapods.org/">CocoaPods</a>) by adding the following line to your <code class="inline-code">Podfile</code>:
```

```
</span>
</li>
</ol>
      </TabPanel>
        <TabPanel>
          <div class="warningBlock">
          This device mode integration is supported for Amplitude v8.8.0 and above.
        </div>
        Follow these steps to add Amplitude to your iOS project:
<ol>

> <li> Install <code class="inline-code">RudderAmplitude</code> by adding the following line to your <code class="inline-code">Podfile</code>.

Reviewer 2: ```suggestion
<li> Install <code class="inline-code">RudderAmplitude</code> (available through [CocoaPods](https://cocoapods.org/)) by adding the following line to your <code class="inline-code">Podfile</code>.
```

<span>

```ruby
pod 'RudderAmplitude', '~> 1.0.0'

# for using IDFA as device id, location listening only

> pod 'Amplitude', '~> 8.8.0'

Reviewer 2: Don't need this line.
